experiment/generate_tests.py

```diff
@@ -22,18 +22,56 @@
 
   $ ./experiment/generate_tests.py \
       --yaml-config-path=experiment/test_config.yaml \
-      --json-config-path=jobs/config.json
+      --json-config-path=jobs/config.json \
+      --prow-config-path=prow/config.yaml
 """
 
 import argparse
 import hashlib
 import json
 import os
-import yaml
+import ruamel.yaml as yaml
```

Comment:
can you use [pyyaml](http://pyyaml.org/) here and you can simply do `import yaml`. We have it in our bazel image so you don't need to worry about failing presubmit here.